views/movies/new.hbs

```diff
@@ -0,0 +1,21 @@
+<h2>Add New Movie</h2>
+
+  <form action="/movies/new" method="GET">
```

Comment:
Here the method should be POST and the url you pass to action should be the same as the POST route you define in movies.js to create a new movie.